### PR TITLE
Colibri cleanup

### PR DESCRIPTION
**Pull request by bgrozev**

### CODE REVIEW

File: src/net/java/sip/communicator/impl/protocol/jabber/extensions/colibri/ColibriBuilder.java [hunk 1]
```diff
@@ -335,66 +319,96 @@ public boolean addAllocateChannelsReq(
...
+    }
+
+    /**
+     * Copies the contents of a {@link RtpDescriptionPacketExtension} to a
+     * {@link ColibriConferenceIQ.Channel}.
+     * @param description the {@link RtpDescriptionPacketExtension} from which
+     * to copy.
+     * @param channel the {@link ColibriConferenceIQ.Channel} to which to copy.
+     */
+    private boolean copyDescription(
```
Comment (bbaldino) [inline src/net/java/sip/communicator/impl/protocol/jabber/extensions/colibri/ColibriBuilder.java]: looks like this doesn't access any member data and could be static?

File: src/net/java/sip/communicator/impl/protocol/jabber/extensions/colibri/ColibriBuilder.java [hunk 2]
```diff
@@ -335,66 +319,96 @@ public boolean addAllocateChannelsReq(
...
+     * {@code false} is returned for all combined requests it makes no sense
      * to send it.
-     * @throws IllegalArgumentException if <tt>localChannelsInfo</tt> does not
-     * describe any endpoints.
      */
     public boolean addBundleTransportUpdateReq(
-        IceUdpTransportPacketExtension    transport,
-        ColibriConferenceIQ               localChannelsInfo)
+            IceUdpTransportPacketExtension transport,
+            String channelBundleId)
```
Comment (bbaldino) [inline src/net/java/sip/communicator/impl/protocol/jabber/extensions/colibri/ColibriBuilder.java]: extra space looks like

File: src/net/java/sip/communicator/impl/protocol/jabber/extensions/colibri/ColibriBuilder.java [hunk 3]
```diff
@@ -1001,28 +980,30 @@ private void assertRequestType(RequestType currentReqType)
...
     /**
-     * Copies transport info from <tt>ContentPacketExtension</tt> to
-     * <tt>ColibriConferenceIQ#ChannelCommon</tt>.
+     * Copies the transport info from a {@link ContentPacketExtension} to a
+     * {@link ColibriConferenceIQ.ChannelCommon}.
+     * @param content the {@link ContentPacketExtension} from which to copy.
+     * @param channel the {@link ColibriConferenceIQ.ChannelCommon} to which to copy.
      */
-    private void copyTransportOnChannel(
+    private void copyTransport(
```
Comment (bbaldino) [inline src/net/java/sip/communicator/impl/protocol/jabber/extensions/colibri/ColibriBuilder.java]: looks like this could be static too

**Comment by bgrozev** [comment]
Thanks, @bbaldino . I've made the three modifications you suggested locally and will include them in the next PR I open since this one is already merged.

**Pull request merged**